lib/dry/configurable/instance_methods.rb

```diff
@@ -11,9 +11,16 @@ module Configurable
     # @api private
     module Initializer
       # @api private
-      def initialize(*)
-        @config = Config.new(self.class._settings.dup)
-        super
+      if RUBY_VERSION >= "2.7"
+        def initialize(*, **)
+          @config = Config.new(self.class._settings.dup)
+          super
+        end
+      else
+        def initialize(*)
+          @config = Config.new(self.class._settings.dup)
+          super
+        end
       end
```

Comment:
To keep compatibility with 2.6, I tried various incantations of this method using `ruby2_keywords` (as suggested on https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/), but I couldn't get any to work.

Given this is a simple method, and given we're likely to drop support 2.6 fairly soon, and that we're not having to do this anywhere else within the lib, I'm OK with the duplication here.

Maybe you might have some ideas about how to handle this better, @flash-gordon?

Reply:
doesn't this work?
```ruby
    module Initializer
      # @api private
      def initialize(*)
        @config = Config.new(self.class._settings.dup)
        super
      end
      ruby2_keywords(:initialize) if respond_to?(:ruby2_keywords, true)
    end
```
I tested locally on 3.0, it passed

Reply:
@flash-gordon it didn't work with the older versions, IIRC

Reply:
@flash-gordon By gum, you're right! I think I was trying it with `*args, **kwargs` and that was messing it up.